Description: Changes the data path to conform with the FHS.
Origin: other
--- a/src/Namen.h
+++ b/src/Namen.h
@@ -23,7 +23,7 @@
 #define TUTORIAL 105
 
 #ifndef _WIN32
-#define PFAD "/usr/share/BillardGL/"
+#define PFAD "/app/share/BillardGL/"
 #endif
 
 #ifndef PFAD
